source/mainnet/conf.py

```diff
@@ -238,6 +228,10 @@
 html_static_path = ["../_static"]
 html_css_files = ["style.css","tippy.css"]
 
+html_css_files = [
+    '../_static/css/style.css',
+]
```

Comment:
I think the line above is already adding this file

Reply:
Yep, I have cleaned it up now.